docs/ibexa_engage/create_campaign.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Create campaign with Ibexa Engage
+edition: experience
+---
+
+# Create campaign
```

Comment:
@julitafalcondusza we should clearly mention, where the user creates campaigns ->Qualifio manager (backoffice)